### PR TITLE
Create Alchemy-Pay-questions-1.md

### LINKED ISSUE #13: Bounty proposal: Compile answers for the 1st set of questions from Alchemy Pay
### Intro

Alchemy Pay https://alchemypay.org/crypto-card is a known Web3 player. 
To proceed they asked some questions.

@pavlovcik please, review and approve this bounty.

### Goal
Compile answers for the 1st set of questions from Alchemy Pay.

### Activity
- Compile answers for the questions
1. Do you require any specialized solutions outside of the standard on/off ramping? (e.g. custom coin listing, custom blockchain, bespoke solutions, etc.)
2. What are some important regions or payment methods that you require?
3. What about the monthly transaction volume you are expecting after integration?
4. When are you looking to integrate?
5. The amounts you would like to release on the customer side?

### Time estimate
1 day

### PR DESCRIPTION
**Pull request by sergfeldman**
Resolves #13

### CODE REVIEW

File: Alchemy-Pay-questions-1.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+**About Ubiquity** 
+
+Ubiquity is a funded startup building highly automated bounties & freelance marketplace, as well as financial services.
```
Comment (pavlovcik) [inline Alchemy-Pay-questions-1.md:3]:
```suggestion
Ubiquity is a venture-backed startup building a highly automated bounties & freelancer marketplace, as well as offering banking services. 
```

File: Alchemy-Pay-questions-1.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+**About Ubiquity** 
+
+Ubiquity is a funded startup building highly automated bounties & freelance marketplace, as well as financial services. 
+
+Our target audience is freelancers - individuals (not businesses) that are located in different countries across the globe. 
+
+We share the Alchemy Pay goal to bridge fiat and crypto global economies seamlessly.
+Ubiquity helps to earn in Web3 and spend everywhere.
```
Comment (pavlovcik) [inline Alchemy-Pay-questions-1.md:8]: I wonder if its beneficial to throw in "metaverse" in these materials.

File: Alchemy-Pay-questions-1.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+**1. Do you require any specialized solutions outside of the standard on/off ramping? (e.g. custom coin listing, custom blockchain, bespoke solutions, etc.)**
+
+We need to issue branded Visa or MasterCard cards for freelancers and perform payouts to these cards.
+
+First of all, we want to perform a quick launch with the basic functionality without specialized solutions.
+API for typical actions will be required for this step. 
+As we understand, Alchemy Pay can provide white-label virtual debit  MasterCard cards branded by Ubiquity.  
+
+In the future, we plan to develop capabilities for suitable DeFi management and advanced security.
```
Comment (pavlovcik) [inline Alchemy-Pay-questions-1.md:20]:
```suggestion
In the future, we plan to develop capabilities for users to be able to manage their "checking accounts" entirely on the blockchain leveraging our stablecoin as their bank balance. 
```

File: Alchemy-Pay-questions-1.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+As we understand, Alchemy Pay can provide white-label virtual debit  MasterCard cards branded by Ubiquity.  
+
+In the future, we plan to develop capabilities for suitable DeFi management and advanced security. 
+Alchemy Pay off-ramp solution and partnership with DeFi protocols can be a good foundation for these new capabilities. 
+
+**2. What are some important regions or payment methods that you require?**
+
+The goal is to provide a global Ubiquity card. 
+We can start with these regions - Asia (ig Singapore and South Korea),  South America (ig Brazil), and Eastern Europe (ig Poland).
+We want to serve countries where freelancers are located, except all jurisdictions included in the FATF black and gray lists, Russia, Belarus.
```
Comment (pavlovcik) [inline Alchemy-Pay-questions-1.md:27]:
```suggestion
We want to serve countries where freelancers are located.
```

I don't think its beneficial to deliberately list this, as they will simply follow their regulations around compliance.

**Comment by pavlovcik** [comment]
Sorry for the late review!

**Comment by sergfeldman** [comment]
> Sorry for the late review!

@pavlovcik These responses were sent to Alchemy Pay immediately after the PR was created.

Thank you for the review. 
I have accepted all the changes, as they will be useful for future communications.

**Pull request merged**